### PR TITLE
sorting entris of sparse matrices

### PR DESCRIPTION
**Pull request by teseoch**

### CODE REVIEW

File: tests/test_sparse_binding.py [hunk 1]
```diff
@@ -60,75 +67,75 @@ def test_dont_clean_up_if_we_reference_members_of_scipy_matrix(self):
...
-        print("  std:", np.std(times_copy))
-        print("  med:", np.median(times_copy))
-
-        print("NOCOPY pybind22:")
-        print("  mean:", np.mean(times_nocopy))
-        print("  std:", np.std(times_nocopy))
-        print("  med:", np.median(times_nocopy))
-
-        self.assertLess(median_nocopy*1e3, median_copy)
+    # def test_timing_for_copy_vs_no_copy(self):
```
Comment (fwilliams) [inline tests/test_sparse_binding.py]: Whoops these are all commented out!

**Review by fwilliams — COMMENTED** [review]
Uncomment the tests! :)

**Comment by teseoch** [comment]
it seems that python 2.6 has a different behaviour.

I vote for dropping the support, also because:
https://pythonclock.org

File: tests/test_sparse_binding.py [hunk 2]
```diff
@@ -17,6 +17,13 @@ def test_sparse_matrix_wrapper(self):
         b = npe_help.sparse_return(a)
         self.assertTrue(a is b)
 
+    def test_sparse_matrix_sorting(self):
+        a = sp.random(200, 200)
+        b = a @ a
```
Comment (fwilliams) [inline tests/test_sparse_binding.py]: do `b.dot(a)` and it will work on Python 2.7.

**Pull request merged**